src/App.tsx

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
 import { Link, useLocation, useRoutes } from 'react-router-dom';
 import TestDemo from './demos/TestDemo';
+import TestComponent from  '../instrumented/components/TestComponent/TestComponent'
```

Comment:
yeah, it's correct, if you check `package.json`, you'll find that the `instrumented` directory is generated from there before running the tests

Reply:
oooh, ok.

I don't think I like this then. The demo app will also be used as a sandbox for development -- but this folder won't exist until tests are run.

Also, this would be you'd have to re-run instrumenting anytime you make a change to components (essentially getting rid of hot reloading?). I think the ideal solution would have the components/hooks under test being instrumented on the fly before testing. 

Was this part of the solution to not being able to get the hooks to be instrumented? If that's the case, maybe we go with this for now (so the project can keep moving forward), but I'll keep researching a solution to that. Because the testing works, which is cool, but this kind of seems like a smell to me.